src/phases/scale/LargeScalePhases.yml

```diff
@@ -236,6 +236,15 @@ SnapshotScanner5GigabytesCmd:
   CollectionSortOrder: forward
   GlobalRate: 1 per 5 minutes
 
+# Leave out the ScanDuration since we'll be supplying the ScanRateMegaBytes option.
+SnapshotScanner5GigabytesFixedRateCmd:
+  Duration: *Duration
+  ScanSizeBytes: 500000000
+  ScanType: snapshot
+  CollectionSkip: 300
+  CollectionSortOrder: forward
+  GlobalRate: 1 per 5 minutes
```

Comment:
What is the global rate here, how does it play along scan rate megabytes ?

Reply:
The GlobalRate is how often the scanner is invoked during a phase. So if it finishes and the phase is still happening due to some other actors, it will start another one in 5 minutes.

Reply:
ok, makes sense.